index.html

```diff
@@ -0,0 +1,270 @@
+              </p>
+            </div>
+            <div class="itemcontainer">
+              <div>
+                <img
+                  src="./images/ccIgnite.svg"
+                  alt="ccIgnite"
+                  class="responsive sectiontwo__container__items__item__responsive__respo"
+                />
+              </div>
+              <span>Forum</span>
+              <p>
+                Have the time to share your thoughts and opinionswith experts
+                for each topic
+              </p>
+            </div>
+            <div class="itemcontainer">
+              <div>
+                <img
+                  src="./images/workshop.svg"
+                  alt="workshop"
+                  class="responsive sectiontwo__container__items__item__responsive__respo"
+                />
+              </div>
+              <span>Workshop</span>
+              <p>
+                Denotaing and small community using open sources and license
+                than sitting
+              </p>
+            </div>
+            <div class="itemcontainer">
+              <div>
+                <img
+                  src="./images/workshop.svg"
+                  alt="workshop"
+                  class="imageitem "
+                />
+              </div>
+              <span>CC Ignite </span>
+              <p>
+                Get opportunity to network the CC affiliates around the world,
+                also after conference
+              </p>
+            </div>
+          </div>
+          <a href="#" id="first"
+            >SEE THE WHOLE PROGRAM</a
+          >
+          <a href="#" id="second">
+            Join Rwanda National conference</a
+          >
+        </div>
+      </section>
+      <!-- end sectiontwo -->
+      <!-- start sectionthree -->
+      <section class="thirdsection pd-y" id="sectionthree">
+        <div class="container">
+          <h3 class="heading">Featured Speakers</h3>
+          <span class="line"></span>
+          <!-- array of objects -->
+          <div class="thirdsectionspeakers"></div>
+        </div>
+      </section>
+      <!-- end sectionthree -->
+      <!-- start sectionfour -->
+      <section class="fourthsection" id="sectionfour">
+        <div class="container">
+          <h3 class="heading" id="greyheading">Partners</h3>
+          <span class="line"></span>
+          <div class="partners">
+              <div
+                class="partneritem"
+              >
+                <img
+                  src="./images/firstcompany.png"
+                  alt="company-1"
+                />
+              </div>
+              <div>
+                <img
+                  src="./images/secondcompany.png"
+                  alt="company-2"
+                />
+              </div>
+              <div>
+                <img
+                  src="./images/thirdcompany.png"
+                  alt="company-3"
+                />
+              </div>
```

Comment:
- I think the images of partners on your mobile website are a bit too big. Can we reduce their sizes(width and height)? I believe they are perfect on desktop screen, so you don't need to alter the way it looks desktop.

| Your partners section  |Behance Partners section |
| :---:   | :---: | 
|![partnersb](https://user-images.githubusercontent.com/99246359/204081139-09308bb6-76cc-45e9-b864-dd17e844da47.PNG)  |![behancptn](https://user-images.githubusercontent.com/99246359/204081180-643c51cf-504d-488b-b9b4-8233ca2f463b.PNG)  |

What do you think about this implementation? 🤗